Lib/calendar.py

```diff
@@ -41,6 +42,18 @@ def __str__(self):
         return "bad weekday number %r; must be 0 (Monday) to 6 (Sunday)" % self.weekday
 
 
+def __getattr__(name):
+    if name in ['January','February']:
+        warnings.warn(f"The '{name}' attribute is going to be deprecated use '{name.upper()}' instead",
+                        DeprecationWarning,
+                        stacklevel=2)
+        if name == 'January':
+            return JANUARY
+        else:
+            return FEBRUARY
+
+    raise AttributeError(f"module {__name__} has no attribute {name}")
```

Comment:
For consistency with the usual exception message:
```suggestion
    raise AttributeError(f"module '{__name__}' has no attribute '{name}'")
```